src/WebJobs.Extensions.DurableTask/RetryOptions.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using DurableTaskCore = DurableTask.Core;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    /// <summary>
+    ///     Contains retry policies that can be passed as parameters to various operations
+    /// </summary>
+    public class RetryOptions
```

Comment:
>RetryOptions [](start = 17, length = 12)

I like the brevity of this name but don't feel too strongly about keeping it. If you feel like the workaround for the conflict of DurableTaskCore.RetryOptions is not a good idea, kindly propose a new name.